common/djangoapps/student/views.py

```diff
@@ -226,37 +227,51 @@ def reverification_info(course_enrollment_pairs, user, statuses):
     return reverifications
 
 
-def get_course_enrollment_pairs(user, course_org_filter, org_filter_out_set):
+def get_course_enrollments(user, course_org_filter, org_filter_out_set):
     """
-    Get the relevant set of (Course, CourseEnrollment) pairs to be displayed on
-    a student's dashboard.
+    Given a user, return a filtered set of his or her course enrollments.
+
+    Arguments:
+        user (User): the user in question.
+        course_org_filter (str): if not None, ONLY courses of this org will
+            be returned. For use in Microsites.
+        org_filter_out_set (list[str]): if course_org_filter is None, then any
```

Comment:
Nit: This name is a bit awkward. How about `org_blacklist` or `orgs_to_exclude`? Or `excluded_orgs`?